docs/product/troubleshooting.md

```
@@ -191,13 +196,39 @@ cd backend && python -m pytest tests\ -v
+# CASE B — the stem is BURIED (e.g. you want 0010 and head is 0030).
+# There is no way to reach it without reverting 0030…0011 first, one `down` at a
+# time, re-reading `status` between each. That is ~20 destructive down-migrations
+# against real data. In dev, rebuilding the database is almost always the right
+# call instead; in prod, do neither without a backup (docs/product/RUNBOOK-backups.md).
 ```
 
-Migrations are forward-only by default. `down` is available per-stem and must be explicit — there is no `down --all`.
+> ⚠️ Running `down` then `up` while the stem you care about is buried undoes and
+> immediately re-applies the HEAD migration and nothing else — it looks like it
+> worked and changes nothing about 0010.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)